src/libinjector/injector.c

```diff
@@ -947,11 +1045,57 @@ event_response_t injector_int3_cb(drakvuf_t drakvuf, drakvuf_trap_info_t* info)
         PRINT_DEBUG("Injected\n");
         injector->rc = 1;
     }
+    else if (INJECT_METHOD_SHELLCODE == injector->method && STATUS_EXEC_OK == injector->status)
+    {
+        PRINT_DEBUG("Shellcode executed\n");
+        PRINT_DEBUG("RAX : 0x%lx (expexted: 0x7)\n", info->regs->rax);
```

Comment:
/r/expexted/expected

Should there be an actual check for rax == 7 before setting rc to 1?

Reply:
No need, actually I can remove this line. It was there to check the return of the particular provided shellcode.